src/internal.c

```diff
@@ -9562,30 +9562,6 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                     #endif
                     }
                 }
-
-            #ifdef HAVE_SECURE_RENEGOTIATION
```

Comment:
This cannot be removed wholesale without an explanation of why it wasn't needed.

Reply:
I restored it this code. I thought it was going to be a problem with the resumes, but it wasn't a problem.